tests/CppUTestExt/MockFailureTest.cpp

```diff
@@ -229,21 +237,35 @@ TEST(MockFailureTest, MockUnexpectedParameterValueFailure)
 TEST(MockFailureTest, MockExpectedParameterDidntHappenFailure)
 {
     call1->withName("foo").withParameter("bar", 2).withParameter("boo", "str");
+    call1->inputParameterWasPassed("bar");
     call2->withName("foo").withParameter("bar", 10).withParameter("boo", "bleh");
     call2->callWasMade(1);
     call2->inputParameterWasPassed("bar");
     call2->inputParameterWasPassed("boo");
-    call3->withName("unrelated");
-    addAllToList();
+    call3->withName("foo").withParameter("bar", 2).withParameter("boo", "blah").withParameter("baa", 0u);
+    call3->inputParameterWasPassed("bar");
+    call4->withName("foo").withParameter("bar", 20);
+    call5->withName("unrelated");
+    addCallsToList(5);
```

Comment:
Since this is used only once and the parameter adds a bit of complexity to the test, could you change this to have a function "addThreeCallsToList" and one that does "addFiveCallsToList" Then remove the if statements from the "addCallsToLists" and let the addFive just call the addThree and then add 2 more.

Reply:
Done